charts/promtail/templates/prometheus-rules.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if and .Values.serviceMonitor.enabled .Values.serviceMonitor.prometheusRule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: {{ template "promtail.fullname" . }}
+{{- if .Values.serviceMonitor.prometheusRule.namespace }}
+  namespace: {{ .Values.serviceMonitor.prometheusRule.namespace | quote }}
+{{- end }}
+  labels:
+    app: {{ template "promtail.name" . }}
+    chart: {{ template "promtail.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
```

Comment:
Why we don't adapt to the Helm standard labels?

https://helm.sh/docs/chart_best_practices/labels/#standard-labels